lib/cosmos/packets/packet_config.rb

```diff
@@ -69,8 +79,6 @@ def initialize
       @telemetry['UNKNOWN']['UNKNOWN'] = Packet.new('UNKNOWN', 'UNKNOWN', :BIG_ENDIAN)
 
       # Used during packet processing
-      @current_target_name = nil
-      @current_packet_name = nil
```

Comment:
After the refactor these didn't seem necessary. I just ask the @current_packet for it's target and packet name.